pkg/kv/kvtest/store.go

```diff
@@ -0,0 +1,253 @@
+package kvtest
```

Comment:
Thanks for this awesome infra test package!

Reply:
We need to have something like that for block adapter - it will help contributors to check new implementations and for us to have one test suite.